src/ngraph/runtime/generic_cpu/node_wrapper.cpp

```diff
@@ -0,0 +1,44 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
don't we have something similar for Serializer? can we reuse/generalize that code?